### PR TITLE
Fixes #3

### LINKED ISSUE #3: Fails to walk the entire Jade AST
symdiff-jade no longer walks the complete AST of a jade file (in fact, it doesn't get past the html tag) anyone due to a change in the structure of the AST. The `walk()` function seems to expect that there is a `nodes` property on every object (and if not it bails). However, the AST syntax for jade now has blocks, which aggregate nodes. If the node has children, it has a block property with a block object containing its children.

So the AST now looks like this:

    { type: 'Block',
      nodes:
       [ { type: 'Doctype', val: 'html', line: 1 },
         { type: 'Tag',
           name: 'html',
           selfClosing: false,
           block: { type: 'Block',
                    nodes:
                      [ [Object],
                        [Object],
                        [Object],
                        [Object],
                        [Object],
                        [Object],
                        [Object],
                        [Object],
                        [Object],
                        [Object] ] },
           attrs: [],
           attributeBlocks: [],
           isInline: false,
           line: 2 } ],
      line: 0 }

I can PR a quick fix for this.

### PR DESCRIPTION
**Pull request by baileyparker**
Updates the code to properly walk the new AST structure as described in #3.

### CODE REVIEW

File: test/symdiff-jade.test.js [hunk 1]
```diff
@@ -1,4 +1,19 @@
-var extract = require('../index');
+var extract  = require('../index'),
+    readFile = require('fs').readFile,
```
Comment (prayerslayer) [inline test/symdiff-jade.test.js]: Would you mind using `readFileSync` here? These tests are so small that speed doesn’t matter yet and I find synchronous code easier to read.

Comment (baileyparker) [inline test/symdiff-jade.test.js]: Done! 2a9b171

**Comment by prayerslayer** [comment]
Awesome, thanks :+1: 

Will merge and release it this (Berlin time) evening.

**Comment by prayerslayer** [comment]
Released as `0.0.4`.

**Pull request merged**